webapp-django/crashstats/crashstats/tests/test_verifyprocessed.py

```diff
@@ -32,61 +38,72 @@ def test_get_entropy(self):
         assert entropy[0] == "000"
         assert entropy[-1] == "fff"
 
-    @mock_s3_deprecated
-    def test_no_crashes(self, boto_helper):
+    def test_no_crashes(self, boto_helper, monkeypatch):
         """Verify no crashes in bucket result in no missing crashes."""
-        boto_helper.get_or_create_bucket(BUCKET_NAME)
+        monkeypatch.setattr(Command, "get_entropy", get_small_entropy)
```

Comment:
This used to go through all 4096 values of entropy. We don't need to do that for the tests, so I finally got around to mocking that to a smaller number.